components/molecule/notification/src/index.js

```diff
@@ -129,9 +130,7 @@ class MoleculeNotification extends Component {
           <div className={`${CLASS}-iconLeft`}>
             <span className={`${CLASS}-icon`}>{icon || ICONS[type]}</span>
           </div>
-          <div className={`${CLASS}-text`}>
-            <span>{text}</span>
```

Comment:
Removing the `<span>` could break styling (or not). Just to be sure this doesn't affect anything.